scripts/build_in_docker.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+#
+# This script is used in the docker container to create a Cura AppImage.
+#
+
+set -e
+
+# Get where this script resides
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+ROOT_DIR="${SCRIPT_DIR}/.."
+
+# Make sure that cura-build-environment is present
+if [[ -z "${CURA_BUILD_ENV_PATH}" ]]; then
+    echo "CURA_BUILD_ENV_PATH is not defined. Cannot find the installed cura build environment."
+    exit 1
+fi
+
+# Make sure that a directory for saving the resulting AppImage exists
+CURA_APPIMAGES_OUTPUT_DIR="${CURA_APPIMAGES_OUTPUT_DIR:-/home/ultimaker/appimages}"
+if [[ ! -d "${CURA_APPIMAGES_OUTPUT_DIR}" ]]; then
+    mkdir -p "${CURA_APPIMAGES_OUTPUT_DIR}"
+fi
+
+# Set up Cura build configuration in environment variables
+export CURA_VERSION_MAJOR="${CURA_VERSION_MAJOR:-4}"
+export CURA_VERSION_MINOR="${CURA_VERSION_MINOR:-1}"
+export CURA_VERSION_PATCH="${CURA_VERSION_PATCH:-99}"
+export CURA_VERSION_EXTRA="${CURA_VERSION_EXTRA:-docker}"
+export CURA_BUILD_NAME="${CURA_BUILD_NAME:-docker}"
+
+export CURA_SDK_VERSION="${CURA_SDK_VERSION:-6.0.0}"
+export CURA_CLOUD_API_ROOT="${CURA_CLOUD_API_ROOT:-https://api.ultimaker.com}"
+export CURA_CLOUD_API_VERSION="${CURA_CLOUD_API_VERSION:-1}"
+export CURA_CLOUD_ACCOUNT_API_ROOT="${CURA_CLOUD_ACCOUNT_API_ROOT:-https://account.ultimaker.com}"
+
+export LIBCHARON_BRANCH_OR_TAG="${LIBCHARON_BRANCH_OR_TAG:-WIP_fix_cura_build_docker}"
```

Comment:
True, but this is less important because the point of the script is to make them configurable (via environment variables). We should provide variables to trigger each build via a job instead of changing the defaults every time we make a new release.